nano/templates/code_mirror.tmpl

```diff
@@ -0,0 +1,14 @@
+<h1>HELLO</h1>
```

Comment:
NanoUI and codemirror won't work.

At all.

For the same reason you never got my chart to function in NanoUI.

HTML interface module for life.